src/Bookings.js

```diff
@@ -1,19 +1,52 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
-
+import SearchResults from "./components/SearchResults.js";
+import fakeBookings from "./data/fakeBookings.json";
 const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+  const [bookings, setBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    console.log("Page loaded");
+    setIsLoading(true);
+    setError(null);
```

Comment:
Really nice use of error handling here on page load. However, you shouldnt need this since you already have the default set to null.